automatic/php/php.nuspec

```diff
@@ -8,7 +8,16 @@
     <authors>PHP Authors</authors>
     <owners>chocolatey, Rob Reynolds</owners>
     <summary>PHP – widely-used general-purpose scripting language that is especially suited for Web development and can be embedded into HTML.</summary>
-    <description>PHP is an HTML-embedded scripting language. Much of its syntax is borrowed from C, Java and Perl with a couple of unique PHP-specific features thrown in. The goal of the language is to allow web developers to write dynamically generated pages quickly.</description>
+    <description>PHP is an HTML-embedded scripting language. Much of its syntax is borrowed from C, Java and Perl with a couple of unique PHP-specific features thrown in. The goal of the language is to allow web developers to write dynamically generated pages quickly.
+
+## Package Parameters
+- `/DontAddToPath` - Do not add install directory to path
+- `/InstallDir:`   - Override the installation directory (needs to be specified both during install and update, until it is remembered by choco)
```

Comment:
maybe - the underlying installer may remember where it is installed for upgrades